### PR TITLE
nydus-image: update documentation docs/nydus-image.md

### PR DESCRIPTION
**Pull request by jiangliu**
Update documentation docs/nydus-image.md.

### CODE REVIEW

File: docs/nydus-image.md [hunk 1]
```diff
@@ -1,65 +1,187 @@
...
 
-Get `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
+## Installation
 
-## Build Nydus Image From Directory Source
+Get latest `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
+## Nydus Image Builder
+
+The `nydus-image create` subcommand creates a RAFS filesystem/Nydus image from a tar file or from a directory.
```
Comment (changweige) [inline docs/nydus-image.md:21]: How about "creates a layer or blob of RAFS filesystem/Nydus image"

**Review by changweige — APPROVED** [review]
Overall LGTM

**Review by adam3q — APPROVED** [review]
LGTM

File: docs/nydus-image.md [hunk 2]
```diff
@@ -1,65 +1,187 @@
...
+### Build RAFS Filesystem in Native Mode from a Directory
 ```shell
-nydus-image create \
-  --bootstrap /path/to/bootstrap \
-  --blob /path/to/blob \
+nydus-image create -t dir-rafs \
+  -D /path/to/output/directory \
   /path/to/source/dir
+  
+[root@image-service]# target/debug/nydus-image create -t dir-rafs -D images/ src
```
Comment (liubin) [inline docs/nydus-image.md:45]: It's better to remove the `target/debug/` prefix for all `nydus-image` commands.

Comment (jiangliu) [reply docs/nydus-image.md:45]: fixed

File: docs/nydus-image.md (hunk 1 above)
Comment (jiangliu) [reply docs/nydus-image.md]: fixed

**Pull request merged**